src/components/context/MyContextProvider.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import MyContext from './MyContext'
+
+class MyContextProvider extends React.Component{
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            showAnswer: false,
+            redirect: true,
+        }
+    }
+
+    toggleAnswer = () => {
+        this.setState(prevState => ({
+            showAnswer: !prevState.showAnswer,
+        }));
+    }
+
+    toggleRedirect = () => {
+        this.setState(prevState => ({
+            redirect: !prevState.redirect,
+        }));
+    }
+
+    render() {
+        const {showAnswer, redirect} = this.state;
```

Comment:
const { showAnswer, redirect } = this.state; <- would be better

Reply:
Done!